trainingMaquetado/Maquetado.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+   
+	<div class= "headline"></div>
+    <title>Cheese Guardian</title>
+    <link rel="stylesheet" href="Maquetado.css">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- This is a comment, by the way -->
+
+</head>
+
+<body>
+
+
+
+<div class="upper-container">
+    <img class= "esthetic" src="photos/phone.png">
```

Comment:
remove the space that separates the `=` from the `"` leave it like this:
`class="esthetic"`